.github/workflows/main.yml

```
@@ -160,3 +160,6 @@ jobs:
           $env:TASK = "${{ matrix.task }}"
           conda init powershell
           & "$env:GITHUB_WORKSPACE/.ci/test_windows.ps1"
+  allgood:
+    runs-on: ubuntu-latest
+    needs: [test]
```

looks like `steps` are required

```suggestion
  allgood:
    runs-on: ubuntu-latest
    needs: [test]
    steps:
    - name: echo allgood
      run: echo "allgood"
```